recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
> I think switching back to an older version just because not to use `svn`.

That is not the reason. The reason is to understand why we need that instead of a stable tarball? (Maybe the release process is too slow? Maybe ArchLinux send patches upstream and decided to use the release that has their patches, etc...)

> ArchLinux already packages `10.69.02` and Fedora packages (depending on distribution version) are between `10.66` and `10.71`. Ubuntu and Debian use their own fork of `netpbm` called `netpbm-free`. The latest tarball release is `10.47`.

They probably had a reason. Let's look into their packing system and find you why. Fedora is usually ver transparent in that regard.

> Don't you think these distributions had a compelling reason to package an `unstable` version as you say?

Probably. Let's find out :wink: